Libero_Projects/import/components/IMC_DGC2/README.md

```diff
@@ -0,0 +1,138 @@
+## Mi-V Extended Subsystem Design Guide Configuration 2: I2C Write & Boot
+This folder contains Tcl scripts that build Libero SoC v2021.3 MIV_ESS DGC2 design project for the Future Avalanche Board. The script is executed in Libero SoC to generate the sample design. 
+
+#### PF_Avalanche_MIV_RV32_BaseDesign (no ES equivalent)
+
+| Config  | Description|
+| :------:|:----------------------------------------|
+| DGC2    | This design uses the **MIV_RV32** core configured as follows: <ul><li>RISC-V Extensions: IMC</li><li>Multiplier: MACC (Pipelined)</li><li>Interfaces: AHB Master (mirrored), APB3 Master</li><li>Reset Vector Address: 0x4000_0000</li><li>Internal IRQs: 6</li><li>TCM: Enabled</li><li>TCM APB Slave (TAS): Enabled</li><li>System Timer: Internal MTIME enabled, Internal MTIME IRQ enabled</li><li>Debug: enabled</li></ul>This design uses the **MIV_ESS** core configured as follows: <ul><li>Bootstrap: Enabled</li><li>Bootstrap Source: I2C</li><li>uDMA: Disabled</li><li>GPIO: Enabled, 2 GPIO_IN and 4 GPIO_OUT (fixed config)</li><li>I2C: Enabled, Two-Byte I2C Address</li><li>PLIC: Disabled</li><li>SPI: Disabled</li><li>Timer: Disabled</li><li>UART: Enabled</li><li>Watchdog: Disabled</li></ul>|
+
+**Note:** This design configuration is only available for the  Polar Fire Avalanche Kit (Revision 3 with production silicon devices). For programming and debugging, this configuration requires the Dual EE Click board to be connected the Avalanche Board mikroBUS headerconnected to the MikroBus header fo
+
+## <a name="quick"></a> Instructions
```

Comment:
Here onwards the sections look all mixed up. Would the below sequence help?

 - Build Libero Design
   - Running Libero SoC in GUI mode, with Script Arguments
     - Script Arguments  
 - Running the DGC2 design on board
   -  Hardware setup requirements 
      - (mention EE MIkrobus etc here.)
   -  Programming .job to the board
      - Instructions here
   -  Run DGC2 firmware as elf.
      - Instructions here
   -  Key pressing sequence
      - Instructions here
   -  Detailed boot sequence 
      - (explain it here)
   -  provided software 
      - (provide links, mention used build configs etc)

Reply:
I ran out of time to make all of the changes, I'll make a note of this one. I might get a chance to include this in while waiting for PR review.